src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -69,6 +69,11 @@
             Enabled="false"
             DisplayName="ms-resource:AppNameDev" />
         </uap5:Extension>
+        <uap3:Extension Category="windows.appExtensionHost">
+              <uap3:AppExtensionHost>
+                <uap3:Name>Microsoft.com.Terminal</uap3:Name>

```

Comment:
That's a weird `uap3:Name` here.... do you mean `com.microsoft.windows.terminal.settings` now?